admin/views/tool-import-export.php

```diff
@@ -418,36 +413,30 @@
 						$option_instance = WPSEO_Options::get_option_instance( $name );
 						if ( is_object( $option_instance ) && method_exists( $option_instance, 'import' ) ) {
 							$optgroup = $option_instance->import( $optgroup, $old_wpseo_version, $options );
-						}
-						elseif ( WP_DEBUG === true || ( defined( 'WPSEO_DEBUG' ) && WPSEO_DEBUG === true ) ) {
+						} elseif ( WP_DEBUG === true || ( defined( 'WPSEO_DEBUG' ) && WPSEO_DEBUG === true ) ) {
 							$msg = sprintf( __( 'Setting "%s" is no longer used and has been discarded.', 'wordpress-seo' ), $name );
 
 						}
 					}
 					$msg = __( 'Settings successfully imported.', 'wordpress-seo' );
-				}
-				else {
+				} else {
 					$msg = __( 'Settings could not be imported:', 'wordpress-seo' ) . ' ' . __( 'No settings found in file.', 'wordpress-seo' );
 				}
 				unset( $options, $name, $optgroup );
-			}
-			else {
+			} else {
 				$msg = __( 'Settings could not be imported:', 'wordpress-seo' ) . ' ' . __( 'Unzipping failed - file settings.ini not found.', 'wordpress-seo' );
 			}
 			@unlink( $filename );
 			@unlink( $p_path );
-		}
-		else {
+		} else {
 			$msg = __( 'Settings could not be imported:', 'wordpress-seo' ) . ' ' . sprintf( __( 'Unzipping failed with error "%s".', 'wordpress-seo' ), $unzipped->get_error_message() );
 		}
 		unset( $zip, $unzipped );
 		@unlink( $file['file'] );
-	}
-	else {
+	} else {
 		if ( is_wp_error( $file ) ) {
 			$msg = __( 'Settings could not be imported:', 'wordpress-seo' ) . ' ' . $file->get_error_message();
-		}
-		else {
+		} else {
 			$msg = __( 'Settings could not be imported:', 'wordpress-seo' ) . ' ' . __( 'Upload failed.', 'wordpress-seo' );
```

Comment:
I should do `$msg = __( 'Settings could not be imported:', 'wordpress-seo' );` outside the if and concat the rest of the string to it in the if-else.